lib/ansible/modules/system/user.py

```diff
@@ -194,6 +194,14 @@
         type: bool
         default: 'no'
         version_added: "2.4"
+    backup:
```

Comment:
i would not even make this an option, specially since it behaves differntly than other 'backup' options, we should just 'do this'

Reply:
Ok. I'll just add a note then and remove the option.